plugins/outputs/remotefile/README.md

```diff
@@ -0,0 +1,78 @@
+# Remote File Output Plugin
+
+This plugin writes telegraf metrics to files in remote locations using the
+[rclone library](https://rclone.org). Currently the following backends are
+supported:
+
+- `local`: [Local filesystem](https://rclone.org/local/)
+- `s3`: [Amazon S3 storage providers](https://rclone.org/s3/)
+- `sftp`: [Secure File Transfer Protocol](https://rclone.org/sftp/)
+
+## Global configuration options <!-- @/docs/includes/plugin_config.md -->
+
+In addition to the plugin-specific configuration settings, plugins support
+additional global and plugin configuration settings. These settings are used to
+modify metrics, tags, and field or create aliases and configure ordering, etc.
+See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
+
+[CONFIGURATION.md]: ../../../docs/CONFIGURATION.md#plugins
+
+## Secret-store support
+
+This plugin supports secrets from secret-stores for the `remote` option.
+See the [secret-store documentation][SECRETSTORE] for more details on how
+to use them.
+
+[SECRETSTORE]: ../../../docs/CONFIGURATION.md#secret-store-secrets
+
+## Configuration
+
+```toml @sample.conf
+# Send telegraf metrics to file(s) in a remote filesystem
+[[outputs.remotefile]]
+  ## Remote location according to https://rclone.org/#providers
+  ## Check the backend configuration options and specify them in
+  ##   <backend type>[,<param1>=<value1>[,...,<paramN>=<valueN>]]:[root]
+  ## for example:
+  ##   remote = 's3,provider=AWS,access_key_id=...,secret_access_key=...,session_token=...,region=us-east-1:mybucket'
+  # remote = "local:"
```

Comment:
In this example of a local write, where would this write to? `/`? the current dir?